packages/pigeon/platform_tests/test_plugin/ios/Classes/CoreTests.gen.swift

```diff
@@ -117,23 +117,23 @@ struct AllNullableTypes {
   var aNullableString: String? = nil
 
   static func fromList(_ list: [Any?]) -> AllNullableTypes? {
-    let aNullableBool = list[0] as? Bool 
-    let aNullableInt = list[1] as? Int32 
-    let aNullableDouble = list[2] as? Double 
-    let aNullableByteArray = list[3] as? FlutterStandardTypedData 
-    let aNullable4ByteArray = list[4] as? FlutterStandardTypedData 
-    let aNullable8ByteArray = list[5] as? FlutterStandardTypedData 
-    let aNullableFloatArray = list[6] as? FlutterStandardTypedData 
-    let aNullableList = list[7] as? [Any?] 
-    let aNullableMap = list[8] as? [AnyHashable: Any?] 
-    let nullableNestedList = list[9] as? [[Bool?]?] 
-    let nullableMapWithAnnotations = list[10] as? [String?: String?] 
-    let nullableMapWithObject = list[11] as? [String?: Any?] 
+    let aNullableBool = list[0] as! Bool? 
+    let aNullableInt = list[1] as! Int32? 
+    let aNullableDouble = list[2] as! Double? 
+    let aNullableByteArray = list[3] as! FlutterStandardTypedData? 
+    let aNullable4ByteArray = list[4] as! FlutterStandardTypedData? 
+    let aNullable8ByteArray = list[5] as! FlutterStandardTypedData? 
+    let aNullableFloatArray = list[6] as! FlutterStandardTypedData? 
+    let aNullableList = list[7] as! [Any?]? 
+    let aNullableMap = list[8] as! [AnyHashable: Any?]? 
+    let nullableNestedList = list[9] as! [[Bool?]?]? 
+    let nullableMapWithAnnotations = list[10] as! [String?: String?]? 
+    let nullableMapWithObject = list[11] as! [String?: Any?]? 
     var aNullableEnum: AnEnum? = nil
     if let aNullableEnumRawValue = list[12] as? Int {
```

Comment:
missing this one?

Reply:
it's not a complete pr, I'm just trying to diagnose the failure of casting to bool? on my local machine. Too see if it's replicated on ci. And perhaps see if there's something obvious from your end

Reply:
The issue was caused by the lists being [Any?] rather than [Any], since [Any] can contain nil, I've decided to change the casting to simplify the process.